helper.fish

```diff
@@ -750,6 +750,22 @@ function buildTarGzPackageHelper
     set name arangodb3
   end
 
+  set -l arch ""
+
+  if test "$USE_ARM" = "On"
+    switch "$ARCH"
+      case "x86_64"
+        set arch "_$ARCH"
+      case '*'
+        if string match --quiet --regex '^arm64$|^aarch64$' $ARCH >/dev/null
+          set arch "_arm64"
+        else
+          echo "fatal, unknown architecture $ARCH for TGZ"
+          exit 1
+        end
+    end
+  end
```

Comment:
This code seems to be duplicated in several places. Can we move it into a function and reduce code duplication?

Reply:
Yes we can do it, but not for this PR.